project/CakeDependencies.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright: 2017 https://github.com/cakesolutions/sbt-cake/graphs
+// License: http://www.apache.org/licenses/LICENSE-2.0
+
+import sbt._
+
+object CakeDependencies {
+
+  // TODO: CO-68: remove JSR305 dependency when SBT moves away from Scala 2.10
+  val jsr305 = "com.google.code.findbugs" % "jsr305" % "3.0.2"
+  val guava = "com.google.guava" % "guava" % "19.0"
+  val config = "com.typesafe" % "config" % "1.3.1"
+  val httpclient = "org.apache.httpcomponents" % "httpclient" % "4.3.6"
+  val leveldbjni = "org.fusesource.leveldbjni" % "leveldbjni" % "1.7"
+  val quasiquotes = "org.scalamacros" %% "quasiquotes" % "2.1.0"
+  val slf4j = "org.slf4j" % "slf4j-api" % "1.7.12"
+
+  object Webjars {
+    val locator = "org.webjars" % "webjars-locator" % "0.26"
+  }
+
+  object Akka {
+    private val version = "2.3.11"
```

Comment:
Lets not make this private. That way we can potential use the version value in other code.